pyproject.toml

```diff
@@ -61,3 +61,23 @@ tensorflow-io-gcs-filesystem = [
     { version = "< 0.32.0", markers = "platform_system == 'Windows'" }
 ]
 resampy = "^0.4.2"
+
+
+[tool.semantic_release]
+version_variable = "pyproject.toml:version"
+tag_format = "v{version}"
+commit_parser = "angular"
+commit_message = "test new message"
+
+
+[tool.semantic_release.branches.main]
+match = "(main|master)"
+prerelease = false
+
+[tool.semantic_release.branches.notmain]
+match = "^(?!main|master)"
+prerelease = true
```

Comment:
Do you want pre-releases?

Reply:
Sure. This will only occur on testing branches for builds. Once we set it up to only run on main, we won't be doing prereleases